pkg/platform/platform.go

```diff
@@ -31,8 +31,9 @@ var cloudProviderConfig = types.NamespacedName{
 }
 
 var (
-	masterRoleMachineConfigLabel = map[string]string{"machineconfiguration.openshift.io/role": "master"}
-	workerRoleMachineConfigLabel = map[string]string{"machineconfiguration.openshift.io/role": "worker"}
+	masterRoleMachineConfigLabel            = map[string]string{"machineconfiguration.openshift.io/role": "master"}
+	workerRoleMachineConfigLabel            = map[string]string{"machineconfiguration.openshift.io/role": "worker"}
+	UserDefinedIPsecMachineConfigAnnotation = map[string]string{"ipsec-machine-config": "true"}
```

Comment:
NIT: should the annotation include some clue about this being a user-defined ipsec configuration?

Reply:
I think @kyrtapz is referring to the key of the annotation itself.

so `user-ipsec-machine-config` vs `ipsec-machine-config`
or `custom-ipsec-machine-config`
or `ipsec-machine-config-override`

Reply:
oh! yes @jcaamano , i missed it. using `user-ipsec-machine-config` as the key for annotation now.